react/week2/to-do-app/src/ToDoList.js

```diff
@@ -0,0 +1,16 @@
+import ToDo from './ToDo';
+
+function ToDoList(props){ 
+    return(
```

Comment:
I would put the add and delete functionality here, as I wrote in the long comment.

Reply:
I have implemented your feedback